CHANGELOG.md

```diff
@@ -0,0 +1,62 @@
+## Unreleased
+
+
+## [v0.1.4](https://github.com/omisego/elixir-omg/releases/tag/v0.1.4) 
+Date: 9/4/19  
+
+- #583 silenced phoenix socket logs
```

Comment:
it seems `#` doesn't expand to a link in markdown - perhaps lets use a full PRs link`https://github.com/omisego/elixir-omg/pull/583` etc? 

Second comment: maybe it's going to be easier and more readable to just scrape the PR titles? If there are typos/Englishos in the PR titles, then perhaps let's amend PR titles :thinking:, so that it is both good-looking and consistent